installer/kickstart-static64.sh

```diff
@@ -124,24 +127,25 @@ run() {
         info_console="[${TPUT_DIM}${dir}${TPUT_RESET}]$ "
     fi
 
+    # shellcheck disable=SC2129
     printf >> "${run_logfile}" "${info}"
     escaped_print >> "${run_logfile}" "${@}"
     printf >> "${run_logfile}" " ... "
 
     printf >&2 "${info_console}${TPUT_BOLD}${TPUT_YELLOW}"
     escaped_print >&2 "${@}"
-    printf >&2 "${TPUT_RESET}\n"
+    printf >&2 "${TPUT_RESET}\\n"
 
     "${@}"
 
     local ret=$?
     if [ ${ret} -ne 0 ]
         then
-        run_failed
-        printf >> "${run_logfile}" "FAILED with exit code ${ret}\n"
+        run_failed "${@}"
+        printf >> "${run_logfile}" "FAILED with exit code ${ret}\\n"
     else
-        run_ok
-        printf >> "${run_logfile}" "OK\n"
+        run_ok "${@}"
```

Comment:
the command is already printed.
these are supposed to be used in case you want to print something:

```
# command to be run is printed here
FAILED   don't worry we can work around this
```